regexes/device/notebooks.yml

```diff
@@ -0,0 +1,114 @@
+###############
+# Device Detector - The Universal Device Detection library for parsing User Agents
+#
+# @link https://matomo.org
+# @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
+###############
+
+Acer:
+  regex: 'FBMD/(?:Aspire E5-421G|Z5WAL|One S1003);'
+  device: 'desktop'
+  models:
+    - regex: 'FBMD/Aspire E5-421G;'
+      model: 'Aspire E5-421G'
+    - regex: 'FBMD/Z5WAL;'
+      model: 'Aspire E5-511'
+    - regex: 'FBMD/One S1003;'
+      model: 'One 10'
+
+Asus:
+  regex: 'FBMD/(?:K50IN|K54L|T100HAN|T103HAF|UX360CAK|X550LB|X553MA|X555LN|X556UQK);'
+  device: 'desktop'
```

Comment:
type notebook?

Reply:
It used to be `notebook` in the original PR, but actually I think that wouldn't be good. We aren't able to say if it's a desktop or notebook in most cases. Having a new notebook type would look like we are able to detect that properly, but actually that only works for few cases...